### PR TITLE
Create OpenStack infrastructure resources

### PR DESCRIPTION
**Pull request by ansd**
Let BBL create OpenStack resources as requested by @voelzmo in https://github.com/cloudfoundry/bosh-bootloader/issues/77.

### CODE REVIEW

File: terraform/openstack/templates/resources-outputs.tf [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+output "internal_ip" {
+  value = "${cidrhost(openstack_networking_subnet_v2.bosh_subnet.cidr, 10)}"
+}
+
+output "router_id" {
+  value = "${openstack_networking_router_v2.bosh_router.id}"
+}
+
+output "director__internal_ip" {
+  value = "10.0.1.6"
```
Comment (voelzmo) [inline terraform/openstack/templates/resources-outputs.tf]: I'm not sure if hardcoding this makes sense, you could also just use terraform's `cidrhost` function to select the `nth` IP from a CIDR: https://www.terraform.io/docs/configuration/interpolation.html#cidrhost-iprange-hostnum-

Comment (ansd) [reply terraform/openstack/templates/resources-outputs.tf]: Done. Thanks.

File: config/load_state_test.go [hunk 2]
```diff
@@ -341,24 +341,21 @@ var _ = Describe("LoadState", func() {
...
+							"--openstack-network-name", "network-name",
 							"--openstack-password", "password",
 							"--openstack-username", "username",
 							"--openstack-project", "project",
 							"--openstack-domain", "domain",
 							"--openstack-region", "region",
-							"--openstack-private-key", "private-key",
+							"--openstack-cacert-file", "/path/to/file",
+							"--openstack-insecure", "true",
+							"--openstack-dns-name-servers", "8.8.8.8,9.9.9.9",
```
[the remark this answers is not in the document]
Comment (ansd) [reply config/load_state_test.go]: done

File: config/merger.go [hunk 3]
```diff
@@ -55,39 +56,25 @@ func copyFlagToStateWithDefault(source string, sink *string, def string) {
 	}
 }
 
+func copyCommaSeparatedFlagToState(source string, sink *[]string) {
```
[the remark this answers is not in the document]
Comment (ansd) [reply config/merger.go]: removed it

**Comment by Polar-Beard** [comment]
Thank you @ansd for your PR! We manually merged it due to some changes we had made with packaging. We really appreciate your work on this (it made our pipelines much smoother 🐯 ).

**Pull request closed without merging**